packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -235,6 +242,10 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
 
         this.messageBuffer.push(message);
 
+        if (message.type === MessageType.NoOp){
+            this.noOpCount++;
```

Comment:
I believe this counter has to be reset in connectHandler(), similar to how we reset this.opsSize in there